src/Dashboard/Types/NodesStatistics/Statistic.php

```diff
@@ -190,8 +201,10 @@ protected function prepareChartData()
      * @param int $length
      * @return string
      */
-    private static function randomString($length = 16) {
+    private static function randomString($length = 16)
```

Comment:
Instead of making a method to create a random string, Laravel already has a build-in method for it:

`str_random()` - https://laravel.com/docs/5.2/helpers#method-str-random